src/graphql/resolvers.ts

```diff
@@ -378,6 +368,17 @@ const Query: QueryResolvers = {
   },
 };
 
+const getExtraInfo = async (slug: string) => {
+  if (slug === "home-banner") {
+    const bannerEnabled = (await getWikiPage("home"))?.content.match(
+      /home_banner_enabled:\W*true/
+    );
+    return { bannerEnabled: false };
```

Comment:
It seems that the `bannerEnabled` constant is not used?

Reply:
Oups, I had hard coded the value for a test 😵‍💫 Thanks.